internal/app/machined/pkg/system/runner/process/process_test.go

```diff
@@ -219,6 +222,148 @@ func (suite *ProcessSuite) TestStopSigKill() {
 	<-done
 }
 
+func (suite *ProcessSuite) TestPriority() {
+	pidFile := filepath.Join(suite.tmpDir, "talos-test-pid-prio")
```

Comment:
do we need to `t.Skip()` this is `os.Getuid() != 0`? (to make `go test ./...` pass without buildkit?)

Reply:
`go test -v ./internal/app/machined/pkg/system/runner/process` works for me